psi4/src/psi4/libscf_solver/rohf.cc

```diff
@@ -494,8 +503,8 @@ void ROHF::form_D() {
 double ROHF::compute_initial_E() { return 0.5 * (compute_E() + nuclearrep_); }
 
 double ROHF::compute_E() {
-    double one_electron_E = Da_->vector_dot(H_) + Db_->vector_dot(H_);
-    double two_electron_E = 0.5 * (Da_->vector_dot(Fa_) + Db_->vector_dot(Fb_) - one_electron_E);
+    double one_electron_E = Dt_->vector_dot(H_);
+    double two_electron_E = 0.5 * (Da_->vector_dot(Ga_) + Db_->vector_dot(Gb_));
```

Comment:
Hmm, is this consistent with the other Wavefunctions. We may need to note somewhere that the external potential gradient is not automatically convolved with the density to provide a density term.